### DISCUSSION TITLE
Samurai wallet DOJO and Whirlpool guide please.

### DISCUSSION THREAD

**Discussion by Lethargy9117** [minibolt-guide/minibolt#71] [Q&A] [answered]
Hi, can you write a guide on Samurai wallet Dojo, whirlpool and whirlpool-CLI installation please?

**Comment by HappyBear-btc** [comment]
agree. the latest Samourai with Sentinel are the best onchain tools for Android available.

**Comment by twofaktor** [comment, marked as answer]
Dojo (without using docker) does not use the `.cookie` RPC authentication method with Bitcoin Core, which means a security risk. The curiosity that an organization focused on privacy (intimately united to security) doesn't consider a priority to integrate this into the Dojo code.

### Context and refs:

https://code.samourai.io/dojo/samourai-dojo/-/issues/160 --> (10 months since they said: "_I will try to find some time to implement this in the future_")

https://x.com/PavelTheCoder/status/1608797010085445632 --> Dojo coder answer: "_If your device gets compromised, you should worry about more things than your RPC creds. Cookie auth adds little to no security in Dojo's case_"

Doc about the importance of using cookie files instead of RPC user/password: https://github.com/Kixunil/security_writings/blob/master/cookie_files.md

As I said: https://twitter.com/twofaktor/status/1603355742056189956: "_It will not be supported in my guide until this is resolved_"
